src/fmt.rs

```diff
@@ -57,4 +57,14 @@ pub fn ask_user_yes_no_question(question: &str) -> io::Result<bool> {
             return Ok(result);
         }
     }
+
+    // This is the dialoguer version of the prompter, but it's not testable
```

Comment:
Have you tried testing it with `rexpect`? Here's an [example](https://github.com/FuelLabs/sway/blob/a81c42eb18827c644f46a6119054d4dacb500f92/forc-plugins/forc-client/tests/deploy.rs#L1349-L1351)